docs/content/en/docs/implementing/tasks-non-k8s-apps.md

```diff
@@ -2,23 +2,46 @@
 title: Keptn for Non-Kubernetes Applications
 description: Using Keptn with Non-Kubernetes Applications
 weight: 95
-hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
 ---
 
-It is possible to trigger Keptn Tasks for workloads and applications that are deployed outside of Kubernetes.
+Keptn Tasks can be triggered for workloads and applications
+that are deployed outside of Kubernetes.
+For example, Keptn could trigger load and performance tests
+for an application that is deployed on a virtual machine.
 
-For example, to trigger a load test for an application deployed on a virtual machine.
+To do this:
 
-You will still need to deploy Keptn on a Kubernetes cluster, but this can be a very lightweight, single-node kind cluster.
-Keptn's only job is to trigger on-demand tasks, so resource utilization will be minimal.
+1. [Install and enable a Kubernetes cluster](#install-and-enable-a-kubernetes-cluster)
+1. [Create a KeptnTaskDefinition](#create-a-keptntaskdefinition)
+1. [Create and apply a KeptnTask](#create-and-apply-a-keptntask)
+1. [Re-run the KeptnTask](#re-run-the-keptntask)
 
-## Step 1: Create a KeptnTaskDefinition
+## Install and enable a Kubernetes cluster
 
-When you have Keptn installed, [create a KeptnTaskDefinition](../implementing/tasks/).
+You must still
+[install](../install/install.md/#use-helm-chart)
+and
+[enable](../install/install.md/#enable-keptn-for-your-cluster)
+Keptn on a Kubernetes cluster,
+but this can be a very lightweight, single-node KinD cluster; see
+[Create local Kubernetes cluster](../install/k8s.md/#create-local-kubernetes-cluster).
+Keptn only triggers on-demand `KeptnTask` resources
+so resource utilization is minimal.
 
-A `KeptnTaskDefinition` defines **what** you want to execute.
+TODO: How is this cluster associated with the VM
+where the deployment is running?
```

Comment:
I don't understand this comment. I think you are trying to ask "how does the VM know it is part of the cluster"? Well you installed Kind on your local machine so that's a single node cluster. Again, this is a detail which kubernetes abstracts from you. You ask it to run a Kubernetes job and hte job is scheduled to a suitable node. You don't (and probably shouldn't) care which node on the cluster. It is the Kubernetes scheduler's job to find you a suitable node.

Reply:
When you say "local machine", do you mean, say, my laptop or the VM/baremetal/whatever machine where the deployment runs?  If Kind is installed on the machine where the deployment runs, this makes sense but this doc doesn't explicitly say that.  We'll talk ;-)

Reply:
Removing this TODO for now.  We can revisit this later.